requirements/README.md

```diff
@@ -0,0 +1,31 @@
+# Requirements files
+
+Dependencies can be installed using either `pip` or `conda`.
+
+## `pip`
+
+* [pip/default.txt](pip/default.txt) &mdash; Default requirements
+* [pip/docs.txt](pip/docs.txt) &mdash; Documentation requirements
+* [pip/test.txt](pip/test.txt) &mdash; Requirements for running tests
+
+#### Example usage
+
+Install the required dependencies for building the documentation.
+
+```bash
+pip install -r requirements/pip/docs.txt
+```
+
+## `conda`
+
+* [conda/default.yml](conda/default.yml) &mdash; Default requirements
+* [conda/docs.yml](conda/docs.yml) &mdash; Documentation requirements
+* [conda/test.yml](conda/test.yml) &mdash; Requirements for running tests
+
+#### Example usage
+
+Install the required dependencies for building the documentation.
+
+```bash
+conda env update --file requirements/conda/docs.yml
```

Comment:
I'm not fully certain how conda handles an update of an environment when the requirements contain both conda and pip packages, but I think this could break things for a user with unknown installed dependencies (e.g. spinx-gallery installed via conda-forge ).